_posts/2022-08-10-#25720.md

```diff
@@ -10,15 +10,53 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- On startup, if a node's headers chain is more than a day behind the current
+  time, it will pick one peer to sync headers with until its best headers chain
+  is caught up. (See
+  [here](https://github.com/bitcoin/bitcoin/blob/92f6461cfd39fff2fc885dd623fa47e7d8d53827/src/net_processing.cpp#L4889-L4913))
+  If a node's headers chain is not caught up within `15 min + ~num headers
+  missing * 1ms` (either because the chosen peer was not able to or because
+  they purposefully stalled the download), it will disconnect the chosen peer
+  for stalling and pick a new peer to sync headers with.
+
+- When a node receives a block announcement via an INV during headers sync, it
+  will add all announcing peers as additional headers sync peers. This week's
+  PR changes this logic such that only one of the announcing peers is added for
+  headers sync.
+
+- [BIP 130](https://github.com/bitcoin/bips/blob/master/bip-0130.mediawiki)
+  (implemented in Bitcoin Core since late 2015) introduced the `sendheaders`
+  message, which indicates that a node prefers receiving block announcements
+  via a `headers` message rather than an `inv`.
+
 ## Questions
-1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
--->
 
+1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or
+   NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
+
+2. Why do nodes (mostly) receive `inv` block announcements while they are doing
```

Comment:
nit: sometimes referred to as "`inv`", sometimes as "INV" - would pick one throughout

Reply:
I prefer lowercase inv as used in pchCommands.
Seems these are all `inv` in the notes now, so resolving.